pyaerocom/aeroval/helpers.py

```diff
@@ -128,17 +148,42 @@ def _get_min_max_year_periods(statistics_periods):
         stop year (may be the same as start year, e.g. if periods suggest
         single year analysis).
     """
-    startyr, stopyr = 1e6, -1e6
+    # startyr, stopyr = 1e16, -1e16
+    startyr, stopyr = start_stop("2100", "1900")
     for per in statistics_periods:
         sl = _period_str_to_timeslice(per)
-        perstart, perstop = int(sl.start), int(sl.stop)
+        # perstart, perstop = sl.start, sl.stop
+        perstart, perstop = start_stop(sl.start, sl.stop)
```

Comment:
I suspect here we can use the same code from main-dev, but it's hard to test without implementing the changes, so I might just do it